config/reader.go

```diff
@@ -561,6 +617,24 @@ func getFromCloudGRPC(ctx context.Context, cloudCfg *Cloud, logger golog.Logger)
 		return nil, shouldCheckCacheOnFailure, err
 	}
 
+	bytes, err := cfg.MarshalJSON()
```

Comment:
Ah ok if you are putting the full config into bytes, might as well convert into map[string]interface{} so that you can find the packages config. Then after you convert back, could you do the replace between the conversion steps, either when the interface is converted back to string or byte[]?